purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -330,14 +304,7 @@ class PostingTransactionsTests {
         val productIds = listOf("uno", "dos")
         val purchase =
             stubGooglePurchase(productIds = productIds).toStoreTransaction(ProductType.SUBS, null)
-        val mockStoreProduct = mockk<StoreProduct>().also {
-            every { it.sku } returns "uno"
-            every { it.priceAmountMicros } returns 2000000
-            every { it.priceCurrencyCode } returns "USD"
-            every { it.subscriptionPeriod } returns ""
-            every { it.introductoryPricePeriod } returns ""
-            every { it.freeTrialPeriod } returns ""
-        }
+        val mockStoreProduct = createStoreProductWithoutOffers("uno")
```

Comment:
These tests still not pass, but they at least compile now. I didn't fix them yet since they are related to posting purchases to the backend, which is a different task.

Reply:
I am doing the same in another branch, commenting what breaks and adding a TODO